### PR TITLE
feat: update fixture and deprecate rinkeby and ropsten

### PR DESCRIPTION
**Pull request by isaackps**
## In this PR:
- Deprecate Ropsten and Rinkeby
- Update all fixtures

### CODE REVIEW

File: scripts/generate.v3.ts [hunk 1]
```diff
@@ -24,10 +24,10 @@ const key = {
 const signMerkleRoot = async (merkleRoot: string) => sign("Secp256k1VerificationKey2018", merkleRoot, key);
 
 const ethereumDocumentConfig = {
-  network: "ropsten",
+  network: "goerli",
   dns: "demo-tradetrust.openattestation.com",
-  documentStore: "0x8bA63EAB43342AAc3AdBB4B827b68Cf4aAE5Caca",
-  tokenRegistry: "0x13249BA1Ec6B957Eb35D34D7b9fE5D91dF225B5B",
+  documentStore: "0x49b2969bF0E4aa822023a9eA2293b24E4518C1DD",
+  tokenRegistry: "0x921dC7cEF00155ac3A33f04DA7395324d7809757",
```
Comment (superical) [inline scripts/generate.v3.ts:30]: ✅

**Review by superical — APPROVED** [review]
Not very familiar with this repo. From what I see, it's mostly changing out the fixtures for the tests to use goerli and updating the demo addresses to the ones on goerli? Let me know if I miss out on anything that I should also look at. Thank you, ser!🙇‍♂️

File: scripts/generate.v3.ts (hunk 1 above)
Comment (isaackps) [reply scripts/generate.v3.ts:30]: Yup! THANK YOU!!!

**Pull request merged**